src/openrct2/object/Object.h

```diff
@@ -251,6 +251,7 @@ class Object
 {
 private:
     std::string _identifier;
+    std::string _version;
```

Comment:
Should there be stored as a string, or be parsed into a major/minor/revision tuple for easy comparison?

Reply:
Thinking about the tuple again, I think if it is parsed from string right-to-left then a 3-entry tuple would be fine, "1.0" that most json objects use would parse as 0.1.0 in major/minor/rev form.

Reply:
You could wrap it in a Version class, but I think it still be kept as an underlying string still - since objects could have anything in that field which might not conform to a major, minor, revision. Also, versions can have a string at the end sometimes, like `2.1.4-preview-13423`.

Reply:
I have just implemented the tuple, it picks numbers from left to right with the . delimiter, so major.minor.rev, major.minor, and major format works. I don't know what expectations there are for comparing random strings.